includes/elements/container.php

```diff
@@ -114,6 +114,7 @@ protected function get_initial_config() {
 	 */
 	protected function content_template() {
 		?>
+		<div class="elementor-edit-click-area"></div>
```

Comment:
Does the click area works also on first Editor load?

Reply:
It does